addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -17,6 +17,11 @@ function animateClone($cart, $elem, offsetTop, offsetLeft) {
                 })
                 .addClass('o_website_sale_animate')
                 .appendTo(document.body)
+                .css({
```

Comment:
what is the wrong effect without it ?

Reply:
An image without fixed size (e.g. in a course page), will take the whole available space when appended to the DOM. which starts the animation with image larger than expected.